src/packages/migrate/src/__tests__/handlePanic.test.ts

```diff
@@ -170,8 +170,6 @@ describe('handlePanic', () => {
       const output = captureStdout.getCapturedText()
       expect(stripAnsi(output.join('\n'))).toMatchInlineSnapshot(`
 
-          console.log    Oops, an unexpected error occured!    Error in migration engine.    Reason: [/some/rust/path:0:0] This is the debugPanic artificial panic        Please create an issue in the migrate repo with    your \`schema.prisma\` and the prisma command you tried to use 🙏:    https://github.com/prisma/migrate/issues/new            Please help us improve Prisma by submitting an error report.    Error reports never contain personal or other sensitive information.    Learn more: https://pris.ly/d/telemetry      at panicDialog (src/utils/handlePanic.ts:24:11)
```

Comment:
@williamluke4 @timsuchanek Any reason this console.log was removed from snapshot? Did you change something in the setup maybe?